search/search-client-elasticsearch/src/main/java/io/camunda/search/es/clients/ElasticsearchSessionDocumentClient.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.es.clients;
+
+import io.camunda.search.security.SessionDocumentStorageClient;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.elasticsearch.action.admin.indices.alias.Alias;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.elasticsearch.common.settings.Settings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ElasticsearchSessionDocumentClient implements SessionDocumentStorageClient {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(ElasticsearchSessionDocumentClient.class);
+  private static final String NUMBER_OF_SHARDS = "index.number_of_shards";
+  private static final String NUMBER_OF_REPLICAS = "index.number_of_replicas";
+  private final RetryElasticsearchClient client;
+  private final String indexName = String.format("%s-%s-%s_", "camunda", "web-session", "1.1.0");
+
+  public ElasticsearchSessionDocumentClient(final RetryElasticsearchClient client) {
+    this.client = client;
+  }
+
+  public void setup() {
+    final Map<String, Object> indexDescription =
+        ElasticsearchJSONUtil.readJSONFileToMap("/camunda-web-session.json");
+    client.createIndex(
```

Comment:
🔧  Should we add try/catch in order to log possible errors over the **_ES operations_** (in general) ?